SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift

```diff
@@ -37,7 +66,40 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
   }
 
   @IBAction func trackEvent(_ : AnyObject) {
-    appCenter.trackEvent(name.stringValue, withProperties: properties)
+    let eventName = name.stringValue
+    for _ in 0..<Int(countSlider.intValue) {
+      if priority != .Default {
+        appCenter.trackEvent(eventName, withProperties: properties, flags: priority.flags)
+      } else {
+        appCenter.trackEvent(eventName, withProperties: properties)
+      }
+    }
+  }
+
+  @IBAction func resume(_ sender: NSButton) {
+    appCenter.resume()
+  }
+
+  @IBAction func pause(_ sender: NSButton) {
+    appCenter.pause()
+  }
+
+  @IBAction func countChanged(_ sender: Any) {
+    self.countLabel.stringValue = "Count: \(Int(countSlider.intValue))"
+  }
+
+  @IBAction func priorityChanged(_ sender: NSComboBox) {
+    switch(self.priorityValue.stringValue)
+    {
+    case "Normal":
```

Comment:
Can't we use the enum for the case values too?  e.g. `case Priority.Normal:`

Reply:
Ok, but it need a "String" type , so I used `case Priority.Normal.rawValue` now.